<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="2" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Open the browser:chrome]]>
    </line>
    <line>
      <![CDATA[Enter the URL:https://demo.actitime.com/login.do]]>
    </line>
    <line>
      <![CDATA[Maximize the browser]]>
    </line>
    <line>
      <![CDATA[Set the TimeOut:10]]>
    </line>
    <line>
      <![CDATA[Close the browser]]>
    </line>
    <line>
      <![CDATA[Open the browser:chrome]]>
    </line>
    <line>
      <![CDATA[Enter the URL:https://demo.actitime.com/login.do]]>
    </line>
    <line>
      <![CDATA[Maximize the browser]]>
    </line>
    <line>
      <![CDATA[Set the TimeOut:10]]>
    </line>
    <line>
      <![CDATA[Close the browser]]>
    </line>
  </reporter-output>
  <suite started-at="2023-02-17T07:59:03 IST" name="Suite" finished-at="2023-02-17T07:59:24 IST" duration-ms="20890">
    <groups>
    </groups>
    <test started-at="2023-02-17T07:59:03 IST" name="Test" finished-at="2023-02-17T07:59:24 IST" duration-ms="20890">
      <class name="scripts.InvalidLogin">
        <test-method is-config="true" signature="openApp(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:scripts.InvalidLogin@72967906]" started-at="2023-02-17T07:59:03 IST" name="openApp" finished-at="2023-02-17T07:59:16 IST" duration-ms="13192" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[https://demo.actitime.com/login.do]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[http://192.168.29.187:4444]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[Open the browser:chrome]]>
            </line>
            <line>
              <![CDATA[Enter the URL:https://demo.actitime.com/login.do]]>
            </line>
            <line>
              <![CDATA[Maximize the browser]]>
            </line>
            <line>
              <![CDATA[Set the TimeOut:10]]>
            </line>
          </reporter-output>
        </test-method> <!-- openApp -->
        <test-method signature="testInvalidLogin()[pri:2, instance:scripts.InvalidLogin@72967906]" started-at="2023-02-17T07:59:16 IST" name="testInvalidLogin" finished-at="2023-02-17T07:59:16 IST" duration-ms="19" status="FAIL">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
Excel cannot be resolved
Excel cannot be resolved
LoginPage cannot be resolved to a type
LoginPage cannot be resolved to a type
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
Excel cannot be resolved
Excel cannot be resolved
LoginPage cannot be resolved to a type
LoginPage cannot be resolved to a type

at scripts.InvalidLogin.testInvalidLogin(InvalidLogin.java:17)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testInvalidLogin -->
        <test-method is-config="true" signature="closeApp()[pri:0, instance:scripts.InvalidLogin@72967906]" started-at="2023-02-17T07:59:16 IST" name="closeApp" finished-at="2023-02-17T07:59:17 IST" duration-ms="584" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Close the browser]]>
            </line>
          </reporter-output>
        </test-method> <!-- closeApp -->
      </class> <!-- scripts.InvalidLogin -->
      <class name="scripts.ValidLogin">
        <test-method is-config="true" signature="openApp(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:scripts.ValidLogin@55182842]" started-at="2023-02-17T07:59:17 IST" name="openApp" finished-at="2023-02-17T07:59:23 IST" duration-ms="6435" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[https://demo.actitime.com/login.do]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[http://192.168.29.187:4444]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[Open the browser:chrome]]>
            </line>
            <line>
              <![CDATA[Enter the URL:https://demo.actitime.com/login.do]]>
            </line>
            <line>
              <![CDATA[Maximize the browser]]>
            </line>
            <line>
              <![CDATA[Set the TimeOut:10]]>
            </line>
          </reporter-output>
        </test-method> <!-- openApp -->
        <test-method signature="testValidLogin()[pri:1, instance:scripts.ValidLogin@55182842]" started-at="2023-02-17T07:59:23 IST" name="testValidLogin" finished-at="2023-02-17T07:59:23 IST" duration-ms="7" status="FAIL">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
Excel cannot be resolved
Excel cannot be resolved
LoginPage cannot be resolved to a type
LoginPage cannot be resolved to a type
ETTPage cannot be resolved to a type
ETTPage cannot be resolved to a type
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
Excel cannot be resolved
Excel cannot be resolved
LoginPage cannot be resolved to a type
LoginPage cannot be resolved to a type
ETTPage cannot be resolved to a type
ETTPage cannot be resolved to a type

at scripts.ValidLogin.testValidLogin(ValidLogin.java:17)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValidLogin -->
        <test-method is-config="true" signature="closeApp()[pri:0, instance:scripts.ValidLogin@55182842]" started-at="2023-02-17T07:59:23 IST" name="closeApp" finished-at="2023-02-17T07:59:24 IST" duration-ms="554" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Close the browser]]>
            </line>
          </reporter-output>
        </test-method> <!-- closeApp -->
      </class> <!-- scripts.ValidLogin -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
